client/lib/products-values/index.js

```diff
@@ -26,6 +26,18 @@ var productDependencies = {
 	}
 };
 
+import {
```

Comment:
it works only because we use Babel to transpile our code :) According to ES6 modules specification, import statement needs to be at the top of the file.